plugin/core/transports.py

```diff
@@ -79,7 +87,6 @@ def _encode(data: Dict[str, Any]) -> bytes:
         return json.dumps(
             data,
             ensure_ascii=False,
-            sort_keys=False,
             check_circular=False,
```

Comment:
Any specific reason for removing this? I think the reason we had it is to not spend time on doing something not strictly needed.

Reply:
[`False` appears to be the default](https://docs.python.org/3.3/library/json.html).